detekt.properties

```diff
@@ -1,14 +1,20 @@
 category=External Analysers
 description=Provide detekt rules for Kotlin projects
 homepageUrl=https://github.com/detekt/sonar-kotlin
-archivedVersions=
-publicVersions=2.0.0
+archivedVersions=2.0.0
+publicVersions=2.1.0
 
 defaults.mavenGroupId=io.github.detekt
 defaults.mavenArtifactId=sonar-detekt
 
+2.1.0.description=Upgrade to detekt 1.9.1
+2.1.0.sqVersions=[7.9,LATEST]
+2.1.0.date=2020-06-02
+2.1.0.changelogUrl=https://github.com/detekt/sonar-kotlin/releases/tag/2.1.0
```

Comment:
As-is, this changelog isn't terribly helpful. Perhaps you could edit your project tag to add the link to the release notes you posted in your community request?

Reply:
Thx, will do!